### ISSUE TITLE
Not adding active to label when using with materializecss

### ISSUE THREAD

**Issue by patrickgordon** [henrychavez/materialize-tags#2]
Hey mate,

Love the little library :+1: 

Got an issue when using with MaterializeCSS (http://materializecss.com/) where when clicking on the field to begin entering tags, it does not add the "active" class to the label and move it off the input line.

Hope this makes sense..

Cheers!

**Comment by henrychavez** [comment]
Hey patrick,

I fixed the problem thanks for report me.

**Comment by patrickgordon** [comment]
Now it doesn't work at all. Getting "Uncaught SyntaxError: Unexpected token
<"

Patrick Gordon
Co-founder @ fitquick.co <http://signup.fitquick.co>
Blog @ medium.com <https://medium.com/@patrickgordon>

Mob: 0433 596 599
Email: patrick@fitquick.co

**Comment by patrickgordon** [comment]
This still occurs for me?

**Comment by henrychavez** [comment]
please go to samples page and donwload de zip with the laest dist.

**Comment by patrickgordon** [comment]
Hmm yep I've downloaded the latest dist. I'm using this with Ruby on Rails with Simple_Form. Anything I can share here that might let us find the cause?

**Comment by patrickgordon** [comment]
OK I found why it is not working for me.

I believe `simple_form` is adding the following classes to my label: 
`<label class="string optional" for="event_category_list">Categories</label>`

As such when this code runs it does not apply active class.

```
            if (typeof label.attr('class') == 'undefined' || label.attr('class') == '')
            {
                label.addClass('active');
            }
```

For our project I just removed the if statement and just let it do:
`label.addClass('active');`

and it works great!
